packages/cfengine/build.sh

```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://cfengine.com/
+TERMUX_PKG_DESCRIPTION="CFEngine is a configuration management technology that helps systems administrators automate and ensure the availability, security and compliance of mission-critical applications and services."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=(3.15.0-2
+                    3.15.0-2)
+TERMUX_PKG_SRCURL=(https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.15.0.tar.gz
+                   https://github.com/cfengine/masterfiles/archive/3.15.0-2-build1.tar.gz)
+                   #https://cfengine-package-repos.s3.amazonaws.com/enterprise/Enterprise-3.15.0/misc/cfengine-masterfiles-3.15.0.pkg.tar.gz)
```

Comment:
Use the version above here, so it can be easily changed, and setting `TERMUX_PKG_REVISION` to 2 if that's changed more often. Remove the last link if you don't need it.